requirements.txt

```diff
@@ -8,8 +8,8 @@ backports.csv; python_version < '3.0'
 Pillow
 six>=1.7.0
 python-dateutil
-social-auth-core>=2.0.0
-social-auth-app-django>=3.0.0
+social-auth-core[openidconnect]>=3.1.0
```

Comment:
I don't mind increasing the required versions, but pulling in openidconnect dependencies unconditionally seems over - most people won't use that. At most that should be in requirements-optional.txt or not listed at all. We don't list additional dependencies for other optional auth backends as well.

Reply:
Sure thing. I am a rookie in python dev, so I pretty sure there are better solutions :) So if we add it to `requirements-optional.txt`, how would I then trigger this in my build?

Reply:
OK, forget about my question - I see it is already installed by default for openshift - which is my choice of deployment.

Will move it there.